packages/@o3r/core/schematics/service/index.ts

```diff
@@ -46,10 +46,10 @@ function ngGenerateServiceFn(options: NgGenerateServiceSchematicsSchema): Rule {
     let packageName = destination;
     if (workspaceProject?.projectType !== 'application') {
       let rec = '..';
-      while (!tree.exists(path.resolve(destination, rec, 'package.json')) && path.resolve(destination, rec) !== '/') {
+      while (!tree.exists(path.join(destination, rec, 'package.json')) && path.resolve(destination, rec) !== '/') {
```

Comment:
```suggestion
      while (!tree.exists(path.join(destination, rec, 'package.json')) && path.join(destination, rec) !== '/') {
```

Reply:
@matthieu-crouzet can't use  "path.join" while comparing with root path as we need absolute path. Path.join returns relative path. Hence using path.resolve

Reply:
But you are comparing with the tree path that is a relative path to the project 
on Windows you will never have a path matching `'/'`

Reply:
Thats correct.. so made changes to accept both "Drive letter" and "/" in todays [commit](https://github.com/AmadeusITGroup/otter/compare/f97502a894f140f7f559a34a173703c218b6bac5..771e53a3031a92b7d84421ec93e74a7123d613c8)

Reply:
Cool.. will take a look prevent it from resolving paths outside projects